arm/deployment/deployment.go

```diff
@@ -0,0 +1,49 @@
+package deployment
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	"github.com/Azure/azure-sdk-for-go/arm/resources/resources"
+)
+
+// BuildDeployment is a helper that creates a resources.Deployment, which can
+// be used as a parameter for a CreateOrUpdate deployment operation.
+// templateFile is a local Azure template.
+// See https://github.com/Azure-Samples/resource-manager-go-template-deployment
+func BuildDeployment(mode resources.DeploymentMode, templateFile string, parameters map[string]interface{}) (deployment resources.Deployment, err error) {
```

Comment:
Since the package name is deployment I think we can change this function name to Build.